src/platform/emotes/emote.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  generateValidator,
+  JSONSchema,
+  ValidateFunction
+} from '../../validation'
+import { Rarity } from '../../dapps/rarity'
+import { I18N } from '../common'
+import { EmoteRepresentation } from './representation'
+import { EmoteData } from './data'
+
+/** @alpha */
+export type Emote = {
+  id: string
+  version: string
+  names: I18N[]
+  descriptions: I18N[]
+  collectionAddress: string
```

Comment:
Couple of questions:
- What is collection address and why is it necessary?
- Are there emotes without collection address?
- Are there off-chain emotes?
- is `id` the URN?

Reply:
NFT Emotes should be part of collections (even mixed with wearables), so it's basically the same type of data described here.
https://github.com/decentraland/common-schemas/blob/10a57e35c94940f6c6a66ef45d102e2099be946a/src/platform/wearables/wearable.ts#L20


An NFT Emote should have the same versability than a wearable, `collectionAddress` in a wearable is not checked for a minimum size so I guess there can be wearables (and therefore emotes) without collection address.